packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -1760,6 +1767,19 @@ export enum StreamViewType {
   KEYS_ONLY = 'KEYS_ONLY'
 }
 
+/**
+ * DynamoDB's table class.
+ *
+ * @see https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.TableClasses.html
+ */
+export enum TableClass {
+  /** Default table class for DynamoDB */
+  STANDARD = 'STANDARD',
+
+  /** table class for DynamoDB that reduces storage costs compared to existing DynamoDB Standard tables */
```

Comment:
Uppercase, and full stop please:

```suggestion
  /** Table class for DynamoDB that reduces storage costs compared to existing DynamoDB Standard tables. */
```